### PR TITLE
Add latex .bib file for citing webassembly

### PR DESCRIPTION
**Pull request by NANASHI0X74**
continuation of https://github.com/WebAssembly/design/pull/1330/

### CODE REVIEW

File: WASM-specs.bib [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@report{WebAssemblyCoreSpecification,
+  title = {{{WebAssembly Core Specification}}},
+  editor = {Rossberg, Andreas},
+  date = {2019-12-05},
+  institution = {{W3C}},
+  url = {https://www.w3.org/TR/wasm-core-1/},
```
Comment (rossberg) [inline WASM-specs.bib:6]: I wonder if we could also point to the PDF somehow, perhaps as a note?

File: WASM-specs.bib [hunk 2]
```diff
@@ -0,0 +1,8 @@
+@report{WebAssemblyCoreSpecification,
+  title = {{{WebAssembly Core Specification}}},
```
[the remark this answers is not in the document]
Comment (NANASHI0X74) [reply WASM-specs.bib]: I guess they should. FTFY

File: WASM-specs.bib (hunk 1 above)
Comment (NANASHI0X74) [reply WASM-specs.bib]: Have a look at the renamed file, does that look good?

File: WASM-specs.bib [hunk 3]
```diff
@@ -0,0 +1,8 @@
+@report{WebAssemblyCoreSpecification,
```
Comment (masterGoverment) [inline WASM-specs.bib]: README.md

**Pull request merged**